packages/@aws-cdk/aws-codepipeline-actions/test/lambda/lambda-invoke-action.test.ts

```diff
@@ -172,18 +174,18 @@ export = {
             },
           ],
         },
-      }));
+      });
+
 
-      test.done();
-    },
+    });
 
-    "assigns the Action's Role with read-write permissions to the Bucket if it has both inputs and outputs"(test: Test) {
+    testFutureBehavior("assigns the Action's Role with read-write permissions to the Bucket if it has both inputs and outputs", s3GrantWriteCtx, App, (app) => {
```

Comment:
General comment. What's your strategy here with testFutureBehavior on v1? Are we proactively dropping tests for "legacy" behavior already, or supposed to have tests for both behaviors, or some other mix? Feels like by converting this test to `testFutureBehavior`, we are no longer testing the "legacy" (read: default) behavior.

Reply:
I don't have a general strategy. It's more applying discretion case-by-case.

In this specific case, the feature flag is changing policy action from `s3:PutObject*` to `s3:PutObject`.
This is quite well tested in the s3 module, so in consuming modules (such as the ones in this PR), I'm only testing the future behaviour. Feels redundant to test the legacy behaviour.